app/cronjob/trailingTrade/step/remove-last-buy-price.js

```diff
@@ -169,15 +171,20 @@ const execute = async (logger, rawData) => {
     return data;
   }
 
-  if (baseAssetQuantity * currentPrice < parseFloat(minNotional)) {
+  //Get symbol config and define last buy remove price threshold, then get the calculated price.
+  const symbolConfiguration = rawData.symbolConfiguration;
```

Comment:
Use destructing assignment, please.

Reply:
If you correctly configure the vscode, it should warn you of this as well.

![image](https://user-images.githubusercontent.com/5715919/120194474-fd76c380-c260-11eb-810f-c044e23fbf6f.png)